.github/workflows/update-openapi.yaml

```diff
@@ -40,22 +40,15 @@ jobs:
       - name: Copy openapi.json
         run: |
           cd registry
-          cp ./common/src/main/resources/META-INF/openapi.json ./app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json
-
-      - name: Make copy with Auth info
-        run: |
-          cd registry/utils/tools
-          mvn clean package
-          mvn exec:java -Dexec.mainClass="io.apicurio.registry.utils.tools.AddOpenApiAuth" -Dexec.args="../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json ../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi-auth.json"
-          mvn exec:java -Dexec.mainClass="io.apicurio.registry.utils.tools.TransformOpenApiForClientGen" -Dexec.args="../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json ../../app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi-gen.json"
+          cp ./common/src/main/resources/META-INF/openapi.json ./app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v3/openapi.json
+          cp ./common/src/main/resources/META-INF/openapi-v2.json ./app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json
 
       - name: Commit Release Version Change
```

Comment:
Ideally we don't need this workflow at all ...

Reply:
See previous explanation.  If we can serve the openapi file(s) from META-INF and have them show up in `/apis` in the browser, that would be great - and then I would agree we should get rid of this workflow.